exercises/concept/vehicle-purchase/.meta/exemplar.R

```diff
@@ -0,0 +1,20 @@
+needs_license <- function(vehicle) {
+  if (vehicle == "car" || vehicle == "truck") {
+    return(TRUE)
+  }
+  FALSE
```

Comment:
```suggestion
  vehicle == "car" || vehicle == "truck"
```
The if / else is unnecessary here.

Reply:
Obviously, yes. I'm embarrassed to have missed that one - clearly having a bad day.